src/drive/payments/index.ts

```diff
@@ -84,6 +85,14 @@ export class Payments {
     return this.client.get<DisplayPrice[]>('/prices', this.headers());
   }
 
+  public getCoupon(): Promise<CouponAvailable> {
+    return this.client.get('/request-coupon', this.headers());
+  }
+
+  public applyCoupon(): Promise<CouponAvailable> {
```

Comment:
Apply coupon response is not correct. Ref here: 
https://github.com/internxt/payments/blob/9c271d0843a511fbbdcaa800977a1290abeb5ccc/src/controller.ts#L169

For the sake of agility, I am going to accept the PR, but correct this please